Integrations/ZeroFox/ZeroFox.py

```diff
@@ -288,7 +290,14 @@ def http_request(method: str, url_suffix: str, params: Dict = None, data: Union[
         else:
             try:
                 if res.status_code not in {200, 201}:
-                    raise Exception('Failure resolving URL.')
+                    # case of wrong credentials
```

Comment:
is it's case of wrong creds, why are we eventually raising `'Failure resolving URL.'`